docs/source/tutorials/07_using_model_hub_nodes.rst

```diff
@@ -0,0 +1,334 @@
+.. include:: /include/substitution.rst
+
+*********************
+Using Model Hub Nodes
+*********************
+
+PeekingDuck has support for external model hub models. You can leverage the model
```

Comment:
Suggest rephrasing intro line as `PeekingDuck has support for models from external model hubs.`
'coz `model hub models` sounds a bit strange.

Reply:
Changed